### PR TITLE
[Docs] Add Animation guide

### PR DESCRIPTION
**Pull request by brentvatne**
WIP, any feedback welcome @ccheever @ide

### CODE REVIEW

File: docs/Animations.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+#### react-tween-state
+
+[react-tween-state](https://github.com/chenglou/react-tween-state) is a
+minimal library that does exactly what its name suggests: it *tweens* a
+value in a component's state, starting at a **from** value and ending at
+a **to** value. This means that it generates the values in between those
+two values, and it sets the state on every `requestAnimationFrame` with
+the intermediary value.
+
+```
```
Comment (ide) [inline docs/Animations.md]: Use blockquoting instead of code formatting

File: docs/Animations.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+#### rebound
+
+[Rebound.js](https://github.com/facebook/rebound-js) is a JavaScript port of
+[Rebound for Android](https://github.com/facebook/rebound). It is
+similar in concept to `react-tween-state`: you have an initial value and
+set an end value, and rebound generates intermediate values that you can
+use for your animation. Rebound is modeled after spring physics. We
+don't provide a duration when animating with springs - that is
+determined by the spring tension, friction, current value and end value
+- all of which can be set on the spring.  Rebound [is used
```
Comment (ide) [inline docs/Animations.md]: The hyphen is rendered as a bullet point

File: docs/Animations.md (hunk 1 above)
Comment (brentvatne) [inline docs/Animations.md]: I did this at first but it is quite ugly when it's red -- it seems like it's a warning. Maybe I need to look at adding a new style to the docs that handle quotes in a better way.

**Comment by brentvatne** [comment]
Merging this now as I think its usable in its current state, but will continue to improve it if necessary

**Pull request merged**